scripts/twitch-proxy/updateApiKey.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+const envFilePath = path.join(`${__dirname}/../../twitch-proxy/`, '.env');
+console.log(envFilePath);
+require('dotenv').config({ path: envFilePath });
+const fetch = require('node-fetch');
+const fs = require('fs');
+const { TWITCH_CLIENT_ID, TWITCH_CLIENT_SECRET } = process.env;
+
+const updateApiKey = async () => {
+  const envFileText = fs.readFileSync(envFilePath, 'utf-8');
+  const url = `https://id.twitch.tv/oauth2/token?client_id=${TWITCH_CLIENT_ID}&client_secret=${TWITCH_CLIENT_SECRET}&grant_type=client_credentials`;
+  const keyObj = await fetch(url, {
+    method: 'POST'
+  })
+    .then(res => res.json())
+    .catch(err => console.log(err));
+
+  // Remove access token line from .env file
+  const regex = /ACCESS\_TOKEN\=[\w\d]*\n?/gm;
+  fs.writeFileSync(envFilePath, envFileText.replace(regex, ''));
+  // Write new access token line to the end of the file
+  fs.appendFileSync(envFilePath, `ACCESS_TOKEN=${keyObj.access_token}\n`);
```

Comment:
```suggestion
  // Write new file with updated token
  fs.writeFileSync(envFilePath, `TWITCH_CLIENT_ID=${TWITCH_CLIENT_ID}
TWITCH_CLIENT_SECRET=${TWITCH_CLIENT_SECRET}
# Expires in ~60 days
ACCESS_TOKEN=${keyObj.access_token}
`);
```
Rather than searching and replacing, this seems a little more straightforward.  What do you reckon?

Reply:
I agree, this is more straightforward. And rebuilding the whole .env file would catch any errors where we change the `TWITCH_CLIENT_ID` and `TWITCH_CLIENT_SECRET`.